ui/component/viewers/videoViewer/internal/plugins/videojs-mobile-ui/plugin.js

```diff
@@ -61,7 +61,7 @@ const onPlayerReady = (player, options) => {
     !player.el_.ownerDocument.querySelector('.bc-iframe')
   ) {
     player.tech_.el_.setAttribute('playsinline', 'playsinline');
-    player.tech_.supportsFullScreen = function() {
+    player.tech_.supportsFullScreen = function () {
```

Comment:
`Optional` 
Can probably modernize it with `() =>` since the file is touched, and squashed into the existing commit

Reply:
check if the last commit is better and didn't break anything like last time

Reply:
![image](https://user-images.githubusercontent.com/64950861/162202657-794a41bd-3f36-43c7-b9a1-7b3ef0bb5484.png)

Is this intended?  If not, and if the file doesn't need to change in the end, can probably skip the modernization suggestion and restore back to original and squash into `Mobile file page, comments and player improvements for rotated landscape view`.